common/models/templates/src/trie-model.ts

```diff
@@ -453,7 +453,24 @@ class Trie {
   lookup(prefix: string): TextWithProbability[] {
     let searchKey = this.toKey(prefix);
     let rootTraversal = this.traverseFromRoot().child(searchKey);
```

Comment:
These should be const as well?

```suggestion
    const searchKey = this.toKey(prefix);
    const rootTraversal = this.traverseFromRoot().child(searchKey);
```